### PR TITLE
ddrescueview: 0.4alpha4 -> 0.4.5

### LINKED ISSUE #195282: Please update ddrescueview
###### Checklist

- [X] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [X] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: ddrescueview

current version: 0.4alpha4 
desired version: [0.4.5](https://sourceforge.net/projects/ddrescueview/files/Test%20builds/v0.4.5/)

###### Notify maintainers

maintainers: @orivej

###### Note for maintainers

Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by iAmMrinal0**
###### Description of changes

closes #195282 

###### Things done

Package update: [0.4.5](https://sourceforge.net/projects/ddrescueview/files/Test%20builds/v0.4.5/)

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/system/ddrescueview/default.nix [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-  version = "${versionBase}${versionSuffix}";
+  version = "0.4.5";
 
   src = fetchurl {
-    name = "ddrescueview-${versionBase}${versionSuffix}.tar.xz";
-    url = "mirror://sourceforge/ddrescueview/ddrescueview-source-${versionBase}~${versionSuffix}.tar.xz";
-    sha256 = "0v159nlc0lrqznbbwi7zda619is5h2rjk55gz6cl807j0kd19ycc";
+    name = "ddrescueview-${version}.tar.xz";
+    url =
+      "mirror://sourceforge/ddrescueview/ddrescueview-source-${version}.tar.xz";
```
Comment (SuperSandro2000) [inline pkgs/tools/system/ddrescueview/default.nix:10]:
```suggestion
    url = "mirror://sourceforge/ddrescueview/ddrescueview-source-${version}.tar.xz";
```

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 195580` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ddrescueview</li>
  </ul>
</details>

File: pkgs/tools/system/ddrescueview/default.nix [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
+stdenv.mkDerivation rec {
   pname = "ddrescueview";
-  version = "${versionBase}${versionSuffix}";
+  version = "0.4.5";
 
   src = fetchurl {
-    name = "ddrescueview-${versionBase}${versionSuffix}.tar.xz";
-    url = "mirror://sourceforge/ddrescueview/ddrescueview-source-${versionBase}~${versionSuffix}.tar.xz";
-    sha256 = "0v159nlc0lrqznbbwi7zda619is5h2rjk55gz6cl807j0kd19ycc";
+    name = "ddrescueview-${version}.tar.xz";
```
Comment (SuperSandro2000) [inline pkgs/tools/system/ddrescueview/default.nix:8]:
```suggestion
```
We don't really need this because the download URL is already nice

**Review by Luflosi — APPROVED** [review]
Built and tested successfully. LGTM.

**Pull request merged**